salt/modules/at.py

```diff
@@ -162,13 +162,19 @@ def atrm(*args):
         salt '*' at.atrm all [tag]
     '''
 
+    import logging
```

Comment:
Is there a reason this import and logger setup is inside the function? We usually put them at the top of the file so it can be used throughout the module.

Reply:
@rallytime Left in from some debugging.  Pushing an update without it.